src/components/Data.jsx

```diff
@@ -0,0 +1,29 @@
+import React, { useContext } from 'react';
+import { useCollection } from 'react-firebase-hooks/firestore';
+import { Form } from './Form';
+import { DataContext } from '../contexts/DataContext';
+
+const Data = () => {
+  const context = useContext(DataContext);
```

Comment:
what do you think to put directly 

`  const { getAll } = useContext(DataContext);`

Reply:
Thank you for your feeback @FauricioValencia  😅😊 we are going to work on those changes